pkg/server/claims_test.go

```diff
@@ -23,26 +23,29 @@ import (
 	"github.com/abcxyz/pkg/testutil"
 )
 
-func TestTokenClaimString(t *testing.T) {
+type TestCase[T any] struct {
```

Comment:
It's best to inline this structure instead in the test.

Reply:
Done